### PR TITLE
fix compound child remove parent try2

### PR DESCRIPTION
**Pull request by BluePrintRandom**
UPBGE: Fix compound child remove parent

previously the fix before worked for the test case but not for my game

youle pointed out something that worked for both

edit *corner case was remove parent -> set parent in same frame

### CODE REVIEW

File: source/gameengine/Ketsji/KX_GameObject.cpp [hunk 1]
```diff
@@ -411,16 +411,14 @@ void KX_GameObject::RemoveParent()
 	m_node->SetLocalScale(m_node->GetWorldScaling());
 	m_node->SetLocalOrientation(m_node->GetWorldOrientation());
 	m_node->SetLocalPosition(m_node->GetWorldPosition());
-	KX_GameObject * parent = static_cast<KX_GameObject *>(m_node->GetParent()->GetObject());
-
-	m_node->DisconnectFromParent();
-	NodeUpdate();
+	
+
```
Comment (marechal-p) [inline source/gameengine/Ketsji/KX_GameObject.cpp]: Remove useless whitespace.

**Comment by marechal-p** [comment]
Also rename the commit to have the prefix and be less weird, maybe something like:
"UPBGE: Follow up fix for compound child remove parent".

**Comment by marechal-p** [comment]
Ok, so you will not get the bonus points today, but the changes look good to me!

**Comment by BluePrintRandom** [comment]
@lordloki  do you mind if I merge?
I tested w/complex and simple files using compound parent

**Comment by marechal-p** [comment]
@BluePrintRandom the message should be the actual commit description.
